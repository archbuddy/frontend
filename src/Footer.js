function Footer() {
  return (
    <div className='footer'>
      All rights reserved to Arch Buddy and team. https://github.com/archbuddy
    </div>
  );
}

export default Footer;