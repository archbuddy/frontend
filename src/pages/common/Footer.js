import { Box, useColorModeValue } from '@chakra-ui/react'

function Footer() {
  return (
    <Box
      fontSize={'small'}
      fontWeight="semibold"
      color="gray.500"
      lineHeight="tight"
      isTruncated
      p={2}
      borderTop={1}
      borderStyle={'solid'}
      borderColor={useColorModeValue('gray.200', 'gray.900')}
    >
      All rights reserved to Arch Buddy and team.{' '}
      <a href="https://github.com/archbuddy" target="_blank" rel="noreferrer">
        https://github.com/archbuddy
      </a>
    </Box>
  )
}

export default Footer
