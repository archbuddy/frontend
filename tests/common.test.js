import renderer from 'react-test-renderer'
import Footer from '../src/pages/common/Footer'

test('render footer', () => {
  const component = renderer.create(<Footer />)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
