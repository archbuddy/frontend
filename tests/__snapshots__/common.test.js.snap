// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render footer 1`] = `
[
  <style
    dangerouslySetInnerHTML={
      {
        "__html": ".css-71vd7t{font-size:small;font-weight:semibold;color:gray.500;line-height:tight;overflow:hidden;text-overflow:ellipsis;white-space:nowrap;padding:2px;border-top:1px;border-style:solid;border-color:gray.200;}",
      }
    }
    data-emotion="css 71vd7t"
  />,
  <div
    className="css-71vd7t"
  >
    All rights reserved to Arch Buddy and team.
     
    <a
      href="https://github.com/archbuddy"
      rel="noreferrer"
      target="_blank"
    >
      https://github.com/archbuddy
    </a>
  </div>,
]
`;
